src/main/scala/org/apache/mesos/chronos/scheduler/mesos/MesosJobFramework.scala

```diff
@@ -73,7 +73,7 @@ class MesosJobFramework @Inject()(
 
   def getReservedResources(offer: Offer): (Double, Double) = {
     val resources = offer.getResourcesList.asScala
-    val reservedResources = resources.filter({ x => x.hasRole && x.getRole != "*" })
+    val reservedResources = resources.filter({ x => x.hasRole && x.getRole != config.mesosRole()})
```

Comment:
This is actually the reverse logic. The original version of this method was correct.